unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/BuilderProjectsPanel.asmdef

```diff
@@ -51,7 +49,9 @@
         "GUID:ee6523961ef177343ad763fcd55c7c46",
         "GUID:f51ebe6a0ceec4240a699833d6309b23",
         "GUID:3b80b0b562b1cbc489513f09fc1b8f69",
-        "GUID:ad97ccc1a866eea46b53413c4254e6cb"
+        "GUID:ad97ccc1a866eea46b53413c4254e6cb",
+        "DeployedScenesFetcher",
+        "LandWIthAccess"
```

Comment:
Why these 2 are without GUID?

Reply:
honestly, I have no idea how that was added and why is mixing GUIDs with names (should be one or another, not both)